app/controllers/application_controller.rb

```diff
@@ -296,4 +296,8 @@ def safe_return_to(url)
       res
     end
   end
+
+  def apiap?
+    current_account.provider_can_use?(:api_as_product)
```

Comment:
this method is already defined in `app/controllers/frontend_controller.rb`, should we redefine here again?

Reply:
No, it is not anymore 😉
https://github.com/3scale/porta/pull/1326/files#diff-1862ca4b36d9904f13fb697bb4daab9eL32-L35

Reply:
👍